### PR TITLE
Drop binaryProvider

### PR DESCRIPTION
**Pull request by amontoison**
@tmigot It should solve your issue with DCISolver.jl.

### CODE REVIEW

**Comment by amontoison** [comment]
@JSOBot runtests

File: src/HSL.jl [hunk 1]
```diff
@@ -7,8 +7,9 @@ using METIS4_jll
 using OpenBLAS32_jll
 using libblastrampoline_jll
 
-if isfile(joinpath(@__DIR__, "..", "deps", "deps.jl"))
-  include("../deps/deps.jl")
+path_deps = joinpath(@__DIR__, "..", "deps", "deps.jl")
+if isfile(path_deps)
+  include(path_deps)
```
Comment (amontoison) [inline src/HSL.jl:12]:
```suggestion
if isfile(joinpath(@__DIR__, "..", "deps", "deps.jl"))
  include("../deps/deps.jl")
```

**Comment by amontoison** [comment]
@JSOBot runtests

**Comment by amontoison** [comment]
@JSOBot runtests

**Review by dpo — APPROVED** [review]
Not very general, but does the job perfectly for now. Thank you!

**Pull request merged**